README.md

```diff
@@ -109,24 +109,29 @@ If you are a library author looking to build OpenTelemetry into your library, pl
 
 ## Supported Runtimes
 
-| Platform Version    | Supported                                       |
-| ------------------- | ----------------------------------------------- |
-| Node.JS `v18`       | ✅                                               |
-| Node.JS `v16`       | ✅                                               |
-| Node.JS `v14`       | ✅                                               |
-| Older Node Versions | See [Node Support](#node-support)               |
-| Web Browsers        | ✅ See [Browser Support](#browser-support) below |
+| Platform Version    | Supported                                     |
+|---------------------|-----------------------------------------------|
+| Node.JS `v18`       | ✅                                             |
+| Node.JS `v16`       | ✅                                             |
+| Node.JS `v14`       | ✅                                             |
```

Comment:
Strange, when I view this page, the check marks come out in two different sizes 🤔: compare the v18 and v16 check-mark sizes:

> <img width="424" alt="image" src="https://github.com/open-telemetry/opentelemetry-js/assets/4140793/f74d96b8-ff38-4b3d-aa8f-35d60a3f2ac9">

Reply:
Hmm, they appeared to have the same size on my machine (Firefox/Ubuntu 22.04) 
Does it look different for you now? I replaced it with GitHub Markdown emojis. [bfe3730](https://github.com/open-telemetry/opentelemetry-js/pull/4037/commits/bfe37301cf18363d090e6ecf23ef4130e7023365)

Reply:
Looks good now! 👍🏻